kotlin/samples/dungeon/common/src/main/java/com/squareup/sample/dungeon/AiWorkflow.kt

```diff
@@ -84,7 +84,7 @@ class AiWorkflow(
       DOWN -> LEFT
       LEFT -> UP
     }
-    println("AI changing direction from ${state.direction} to $newDirection")
+//    println("AI changing direction from ${state.direction} to $newDirection")
```

Comment:
`//`?

Reply:
It's SO NOISY, and since it's not Timber I can't make it `.v` or whatever.

@zach-klippenstein, your call on what I should really do with it. Happy to add Timber.

Reply:
Just delete this logging, it is redundant with the diagnostic listener stuff anyway.